test/mpi/util/mtest_thread_abt.h

```diff
@@ -193,16 +193,27 @@ void MTest_init_thread_pkg(void)
 
     ABT_init(0, NULL);
 
+    /* Get the main pool of the primary execution stream.
+     *
+     * Some threads (e.g., asynchronous progress threads) can be associated with
```

Comment:
Add -- "when MPI_Init_thread is called before `MTest_init_thread_pkg` e.g. `threads/spawn/th_taskmanager`.

Reply:
I updated the comment.